src/text.md

```diff
@@ -109,11 +109,15 @@ Two-factor authentication (sometimes called Multi-Factor Authentication, or MFA)
+If you have an ATM card, you're already using 2FA: the card is something you *have*, and your PIN is something you *know*. If your house has an alarm, that's 2FA too: you *have* your key, and you *know* your alarm code. In fact, something you *know* plus something you *have* is the most common form of 2FA. For online accounts, the thing you *know* will be your password, and the thing you *have* will be your phone, where, after logging in with your username and password, you'll be asked to enter a code either from an authenticator app or from a text message that the service sends you.
 
-When setting up 2FA, if you have the choice, we recommend using an authenticator app over text messages (but if you can only choose text messages, that's fine too). The system used by authenticator apps for 2FA is a standard, so any of the major authenticator apps will work here: LastPass, 1Password, Google, Microsoft, and Authy all make compatible apps, and Aegis gets a bonus point for being open source. Pick one and use it for all of your 2FA-enabled services.
+The system used by authenticator apps for 2FA is a standard, so any of the major authenticator apps will be compatible with it: [Google Authenticator](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&hl=en_US&gl=US), [Microsoft Authenticator](https://www.microsoft.com/en-us/account/authenticator), [Authy](https://authy.com/), and [Aegis](https://getaegis.app/) (which gets a bonus point for being open source) will all work exactly the same way. Pick one and use it for all of your 2FA-enabled services. 1Password and LastPass build a 2FA app right into their password managers, so if you're already using one of them, you don't need an extra app. Because the system for 2FA apps is a standard, even if a service specifically says it uses (for example) Google Authenticator, you can use any one of these apps and it'll work just fine.
 
-You'll also get a list of recovery codes when you set up 2FA. You won't be able to access them again later, so keep them safe and in a place where you can get to them without needing to log into an account that might require a recovery code. Storing printed copies in a safe is another good option.
+When setting up 2FA, if you are given the choice between using an authenticator app or receiving text messages, we recommend using the authenticator app.[^authapp] If the service can only send text messages, that's fine too. The authenticator app strategy is slightly more secure, but not enough that we're going to make a big deal out of it.
+
+  [^authapp]: Google also has a two-factor authentication system where, if you have an Android phone, it will pop up a screen on your phone asking if you're signing in. This only works for Google apps, but it's also a valid choice if you want to use it.
```

Comment:
Apple has a similar service that pops up a code on iOS devices and Macs when you log into Apple services.